packages/wrangler/src/index.tsx

```diff
@@ -164,7 +154,7 @@ export function getScriptName(
 /**
  * Alternative to the getScriptName() because special Legacy cases allowed "name", and "env" together in Wrangler1
  */
-function getLegacyScriptName(
+export function getLegacyScriptName(
```

Comment:
Not worth blocking this PR, but this should probably be moved out of `index.tsx`

Reply:
Absolutely agree, will do a clean-up PR after